src/_data/sponsors.yml

```diff
@@ -23,3 +23,8 @@
   logo_url: /images/sponsors/oivan.svg
   url: "https://oivan.com/"
   level: gold
+- name: PRODIGY9
+  about: "Better people, better software."
+  url: https://www.cloud66.com/frameworks/rails/?utm_source=e&utm_medium=l&utm_campaign=RubyConfTH
+  logo_url: /images/sponsors/prodigy9.svg
+  level: silver
```

Comment:
Do you want to add the missing newline Github is complaining about?

Reply:
Where is it complaining?

Reply:
Right at the end of the file in the diff view:

<img width="224" alt="Screen Shot 2022-10-14 at 23 43 10" src="https://user-images.githubusercontent.com/47985/195898783-451390a7-f576-4a00-9173-22bdf09b0de5.png">

Reply:
I did the simple fix